p4rt_app/tests/critical_state_test.cc

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020 Google LLC
```

Comment:
Can first time addition of a new file be done in a separate PR ?

Reply:
Any new file addition should include the corresponding change in the BUILD file under this dir and the tests run using bazel test cmd.